contracts/DependencyRegistryV0.sol

```diff
@@ -0,0 +1,661 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.17;
+
+// Created By: Art Blocks Inc.
+
+import "./interfaces/0.8.x/IAdminACLV0.sol";
+import "./interfaces/0.8.x/IGenArtDependencyConsumer.sol";
+
+import "@openzeppelin-4.7/contracts/utils/Strings.sol";
+import "@openzeppelin-4.7/contracts/access/Ownable.sol";
+import "@openzeppelin-4.7/contracts/utils/structs/EnumerableSet.sol";
+import "./libs/0.8.x/BytecodeStorage.sol";
+import "./libs/0.8.x/Bytes32Strings.sol";
+
+/**
+ * @title Art Blocks Dependency Registry, V0.
+ * @author Art Blocks Inc.
+ * @notice Privileged Roles and Ownership:
+ * Permissions managed by ACL contract
+ */
+contract DependencyRegistryV0 is Ownable {
+    using BytecodeStorage for string;
+    using BytecodeStorage for address;
+    using Bytes32Strings for bytes32;
+    using Strings for uint256;
+    using EnumerableSet for EnumerableSet.Bytes32Set;
+
+    uint8 constant AT_CHARACTER_CODE = uint8(bytes1("@")); // 0x40
```

Comment:
Nit: could we just call this `AT_CHARACTER`?

Reply:
AH–yeah let's use consistency here. That is my bad :) 

> If we wanted to shove the whole "verify a single @" logic in an expanded version of the existing library being used, that might be a good pattern since we might use this across many contracts in the future.


@ryley-o sorry what library do you mean here?

Reply:
>@ryley-o sorry what library do you mean here?

@jakerockland We already use a AB-created Bytes32Strings library to call the `containsExactCharacterQty` function, so was thinking we could add a function to that library called `containsExactlyOneAtCharacter` or something. Makes the library less generalized, but if we are doing this across multiple contracts, might be helpful/cleaner.

Reply:
Ah I see–I think that is an overly specific thing to add to a generalized `Bytes32Strings` library personally. 

Having a specific constant exist across multiple contracts I think is a bit more idiomatic, easy to follow than pushing that into an otherwise generalized library.

Reply:
sounds good, resolving this one 🥳